src/Uno.UI/UI/Xaml/UIElement.iOS.cs

```diff
@@ -63,12 +63,7 @@ internal bool IsArrangeDirtyPath
 
 		partial void ApplyNativeClip(Rect rect)
 		{
-			if (rect.IsEmpty
-				|| double.IsPositiveInfinity(rect.X)
-				|| double.IsPositiveInfinity(rect.Y)
-				|| double.IsPositiveInfinity(rect.Width)
-				|| double.IsPositiveInfinity(rect.Height)
-			)
+			if (!rect.IsFinite)
```

Comment:
Will this also check for `Rect.Empty` ?

Reply:
it will hit:
```
	public static Rect Empty { get; } = new Rect
	{
		X = double.PositiveInfinity,
		Y = double.PositiveInfinity,
		Width = double.NegativeInfinity,
		Height = double.NegativeInfinity
	};
```